BEGiN !!!

coder-raksh2509 opened the pull request:
__1__ 

<!-- Add the issue number that is fixed by this PR (In the form Fixes #123) -->
<!-- Add issue numbers both above and below this comment, do not remove __ or #-->

Fixes #1

#### Short description of what this resolves:



#### Changes proposed in this pull request and/or Screenshots of changes:

-
-
-

coder-raksh2509 committed:
I AM SUPEREXCITED git add data/participants.jsongit add data/participants.json

coder-raksh2509 committed:
ISSUE NUMBER #123

ekanban commented:
mention issue number

coder-raksh2509 commented:
> mention issue number

SIR  mentioned

coder-raksh2509 committed:
Delete pull_request_template.md

coder-raksh2509 committed:
MAYBE CORRECt

coder-raksh2509 committed:
MAYBE CORRECT

coder-raksh2509 commented:
> mention issue number

I am bit confused ,am I doing it correctly as last time I deleted one of the file but now I have restored it .....AM I cORRECT?

thesmallstar commented:
@coder-raksh2509  you don't need to edit that file.
Just correct the particiapant.json for now, PR template appears when you make a PR, you need to edit it at that point.

coder-raksh2509 commented:
> @coder-raksh2509 you don't need to edit that file.
> Just correct the particiapant.json for now, PR template appears when you make a PR, you need to edit it at that point.

@thesmallstar SO I need to make one more commit ..... restoring the file as the older one na ?

coder-raksh2509 committed:
Revert "I AM SUPEREXCITED git add data/participants.jsongit add data/participants.json"

This reverts commit cff7e4934d21278522c7b499c6ebe97d86188608.

coder-raksh2509 commented:
@thesmallstar  I hope this is fine now

abhishekmsingh commented:
@coder-raksh2509 leave the pull_request_template.md file as it is. Just add your entry in the participants.json. It seems that you have done quite the opposite. Do not edit the pull_request_template.md file.

coder-raksh2509 commented:
> @coder-raksh2509 leave the pull_request_template.md file as it is. Just add your entry in the participants.json. It seems that you have done quite the opposite. Do not edit the pull_request_template.md file.

I have corrected sir

coder-raksh2509 commented:
> > @coder-raksh2509 leave the pull_request_template.md file as it is. Just add your entry in the participants.json. It seems that you have done quite the opposite. Do not edit the pull_request_template.md file.
> 
> I have corrected sir


@thesmallstar @abhiandthetruth 
CAN I BE Helped........I think I have corrected the issues

abhishekmsingh commented on .github/pull_request_template.md:
```diff
@@ -1,12 +1,12 @@
-__ISSUENUMBER__ 
```
You don't need to edit pull_request_template.md file @coder-raksh2509. Please just write your details in participants.json.

Jigar3 commented:
@abhiandthetruth You still are editing the wrong file.
